backend/app/routers/tasks.py

```
@@ -0,0 +1,55 @@
+        raise HTTPException(
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+            detail=f"Invalid priority '{task.priority}'. Must be one of: {sorted(VALID_PRIORITIES)}."
+        )
+
+    try:
+        db_task = models.Task(**task.model_dump())
+        db.add(db_task)
+        db.commit()
+        db.refresh(db_task)
+        return db_task
+    except SQLAlchemyError as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)